### ISSUE TITLE
cannot receive image

### ISSUE THREAD

**Issue by k-okada** [ros-naoqi/naoqi_driver#22]
I had following error message on pepper robot
```
front_camera Camera Handle is empty - cannot retrieve image
infrared_camera Camera Handle is empty - cannot retrieve image
front_camera Camera Handle is empty - cannot retrieve image
depth_camera Camera Handle is empty - cannot retrieve image
infrared_camera Camera Handle is empty - cannot retrieve image
front_camera Camera Handle is empty - cannot retrieve image
depth_camera Camera Handle is empty - cannot retrieve image
front_camera Camera Handle is empty - cannot retrieve image
infrared_camera Camera Handle is empty - cannot retrieve image
```

**Comment by Karsten1987** [comment]
It's a known issue. For me this indicates a problem in naoqi ALVideoDevice.
This error message occurs directly when the proxy towards the ALVideoDevice doesn't deliver a correct image.

Rebooting naoqi or closing the cameras with the ALVideoDevice API may help here.

**Comment by vrabaud** [comment]
a warning message explaining the status and what to do should be written then

**Comment by Karsten1987** [comment]
fixed in 537716962e467985178b7f37cf6534cc0835d1ae
